File: Segundo Corte /Areas_y_Perimetros/Java/Rectangle.java
public class Rectangle {
  
    public double ancho;
    public double largo;
    public Rectangle(double ancho, double largo){
        System.out.println("Creating rectangle with widht "+ ancho + " and length "+ largo);
        this.ancho= ancho;
        this.largo = largo;
    }
    public double Perimetro(){
    return 2*this.largo + 2*this.ancho;
}
    public double Area(){
        return this.largo*this.ancho;
    }
    public static void main(String[] args) {
        Rectangle box1 = new Rectangle(10,13);
        Rectangle box2 = new Rectangle(12,46);
        Rectangle box3 = new Rectangle(14,28);
        
        System.out.println(box1.Perimetro());
        System.out.println(box2.Perimetro());
        System.out.println(box3.Perimetro());
        
        System.out.println(box1.Area());
        System.out.println(box2.Area());
        System.out.println(box3.Area());
    }
        
  }

File: Segundo Corte /Areas_y_Perimetros/Python/Rectangle.py
class Rectangle:
    def _init_(self, largo, ancho):
        print("Creating rectangle with widht {a}".format(a=ancho), " and length {l}" .format(l=largo))
        self.largo = largo
        self.ancho = ancho
    def Perimetro(self) :
        return 2*self.largo + 2*self.ancho
    def area(self):
        return self.largo*self.ancho
    
box1 = Rectangle(10,43)
box2 = Rectangle(12,23)
box3 = Rectangle(16,41)
        
print(box1.Perimetro())
print(box2.Perimetro())
print(box3.Perimetro())

print(box1.area())
print(box2.area())
print(box3.area())
